src/modules/taskListFunctions.js

```diff
@@ -3,11 +3,12 @@ export function addTaskToList(taskList, taskText, index) {
   const li = document.createElement('li');
   const newIndex = taskList.children.length + 1;
   li.innerHTML = `
```

Comment:
- [OPTIONAL] This change is related to the styling of your application. The delete button for the application is outside the parent container. Kindly style it properly. 👍 😎